angr/__main__.py

```diff
@@ -3,22 +3,23 @@
 from angr.analyses.decompiler.structuring import STRUCTURER_CLASSES
 from angr.analyses.decompiler.utils import decompile_functions
 
-COMMANDS = [
-    "decompile",
-]
+
+class COMMANDS:
```

Comment:
Is this supposed to be an enum?

Reply:
@twizmwazin it would be an enum if I knew how to use enums :)

Reply:
I only do this way because I don't like comparing against Enums, instead I like the direct string compare for the class properties

Reply:
I think it's ok before we can use StrEnum after dropping Python 3.8 support.